### PR TITLE
Add "add bulma" recipe

### PR DESCRIPTION
**Pull request by Larocceau**

### CODE REVIEW

File: docs/recipes/ui/add-bulma.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# How do I add Bulma to a SAFE project?
+
+[Bulma](https://bulma.io/documentation/) is a free open-source UI framework based on flex-box that helps you create modern and responsive layouts.
```
Comment (mattgallagher92) [inline docs/recipes/ui/add-bulma.md:3]:
```suggestion
[Bulma](https://bulma.io/documentation/) is a free open-source UI framework based on flexbox that helps you create modern and responsive layouts.
```

File: docs/recipes/ui/add-bulma.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+When using Feliz (the standard for a SAFE app), follow the instructions below. When using Fable.React, use the [Fulma](https://fulma.github.io/Fulma/) wrapper for Bulma.
+
+## 1. Add the Feliz.Bulma NuGet package to the solution
```
Comment (mattgallagher92) [inline docs/recipes/ui/add-bulma.md:7]: I'm not sure that it's really correct to say that it's being added to the solution - the solution is a dotnet concept for gathering related projects. As far as I'm aware, Paket doesn't use the solution concept.

```suggestion
## 1. Add the Feliz.Bulma NuGet package to the client project
```

File: docs/recipes/ui/add-bulma.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+```
+dotnet paket add Feliz.Bulma -p Client
```
Comment (mattgallagher92) [inline docs/recipes/ui/add-bulma.md:10]: This command will only work if executed from within `./src` right (let me know if I'm wrong)? If the location to execute a command from is not specified, I would assume the repo root.

```suggestion
dotnet paket add Feliz.Bulma -p src/Client
```

**Review by mattgallagher92 — CHANGES_REQUESTED** [review]

File: docs/recipes/ui/add-bulma.md (hunk 3 above)
Comment (Larocceau) [reply docs/recipes/ui/add-bulma.md:10]: For me, specifying projects by name instead of path works. I'd say it's cleaner that way right?

Comment (Larocceau) [reply docs/recipes/ui/add-bulma.md:10]: Even from the project root by the way

**Review by mattgallagher92 — APPROVED** [review]

**Pull request merged**